src/t8_forest.h

```diff
@@ -120,6 +121,7 @@ typedef int         (*t8_forest_adapt_t) (t8_forest_t forest,
                                           t8_locidx_t which_tree,
                                           t8_locidx_t lelement_id,
                                           t8_eclass_scheme_c * ts,
+                                          int is_family,
```

Comment:
```suggestion
                                          const int is_family,
```
is_family should not be changed during the adapt-callback, hence it should be const

Reply:
That's right! But isn't that also the case for num_elements, so both should be const?

Reply:
Yes, you are right! Can you change it too?

Reply:
done